demo/allegro5/nuklear_allegro5.h

```diff
@@ -98,6 +98,7 @@ NK_API void nk_allegro5_del_image(struct nk_image* image)
 static float
 nk_allegro5_font_get_text_width(nk_handle handle, float height, const char *text, int len)
 {
+    (void)height;
```

Comment:
Not sure if this is worth doing, but there is a macro we could use for it :wink:
``` suggestion
    NK_UNUSED(height);
```

Reply:
Ooh, I've forgotten about that one. Thanks!